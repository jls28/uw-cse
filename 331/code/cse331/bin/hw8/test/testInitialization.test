# Verify that the program initializes and can quit

# All of your test scripts should end with q to terminate the program.
q