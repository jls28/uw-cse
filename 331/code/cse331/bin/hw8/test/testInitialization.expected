Menu:
r to find a route
b to see a list of all buildings
q to quit

Enter an option ('m' to see the menu): # Verify that the program initializes and can quit

# All of your test scripts should end with q to terminate the program.
